src/plugins/cosmos/components/modals/Staking/Staking.tsx

```diff
@@ -39,9 +39,25 @@ import { UnstakeBroadcast } from './views/UnstakeBroadcast'
 import { UnstakeConfirm } from './views/UnstakeConfirm'
 
 const StakingModalContent = ({ assetId, validatorAddress }: StakingModalProps) => {
+  const [validateAllSteps, setValidateAllSteps] = useState(false)
   const location = useLocation()
   const translate = useTranslate()
 
+  const isBroadcast = matchPath(location.pathname, {
+    path: [StakingPath.Broadcast, ClaimPath.Broadcast, UnstakingPath.Broadcast],
+    exact: true,
+  })
+
+  useEffect(() => {
+    if (!isBroadcast) {
+      setValidateAllSteps(false)
+    }
+  }, [isBroadcast])
+
+  const onValidatedBroadcast = (value: boolean) => {
```

Comment:
Ditto `validated` terminology, that's more like `onBroadcastSuccesful` (it is fired after a succesful broadcast Tx in `<ClaimBroadcast />`